vendor.mod

```diff
@@ -0,0 +1,197 @@
+// 'vendor.mod' enables use of 'go mod vendor' to managed 'vendor/' directory.
+// There is no 'go.mod' file, as that would imply opting in for all the rules
+// around SemVer, which this repo cannot abide by as it uses CalVer.
+
+module github.com/docker/docker
+
+go 1.17
+
+require (
+	cloud.google.com/go v0.81.0
+	cloud.google.com/go/logging v1.4.2
+	github.com/Graylog2/go-gelf v0.0.0-20191017102106-1550ee647df0
+	github.com/Microsoft/go-winio v0.5.1
+	github.com/Microsoft/hcsshim v0.8.23
+	github.com/RackSec/srslog v0.0.0-20180709174129-a4725f04ec91
+	github.com/armon/go-radix v0.0.0-20180808171621-7fddfc383310
+	github.com/aws/aws-sdk-go v1.28.11
+	github.com/bsphere/le_go v0.0.0-20170215134836-7a984a84b549
+	github.com/cloudflare/cfssl v0.0.0-20180323000720-5d63dbd981b5
+	github.com/containerd/cgroups v1.0.1
+	github.com/containerd/containerd v1.5.8
+	github.com/containerd/continuity v0.1.0
+	github.com/containerd/fifo v1.0.0
+	github.com/containerd/typeurl v1.0.2
+	github.com/coreos/go-systemd/v22 v22.3.2
+	github.com/creack/pty v1.1.11
+	github.com/deckarep/golang-set v0.0.0-20141123011944-ef32fa3046d9
+	github.com/docker/distribution v2.7.1+incompatible
+	github.com/docker/docker/autogen/winresources/dockerd v0.0.0-00010101000000-000000000000
+	github.com/docker/go-connections v0.4.0
+	github.com/docker/go-events v0.0.0-20190806004212-e31b211e4f1c
+	github.com/docker/go-metrics v0.0.1
+	github.com/docker/go-units v0.4.0
+	github.com/docker/libkv v0.2.2-0.20211217103745-e480589147e3
+	github.com/docker/libtrust v0.0.0-20150526203908-9cbd2a1374f4
+	github.com/docker/swarmkit v1.12.1-0.20210726173615-3629f50980f6
+	github.com/fluent/fluent-logger-golang v1.8.0
```

Comment:
Note to self: need to update this to v1.9.0 (see https://github.com/moby/moby/pull/43100)